sympy/logic/boolalg.py

```diff
@@ -2769,6 +2769,13 @@ def simplify_logic(expr, form=None, deep=True, force=False):
         made. By setting ``force`` to ``True``, this limit is removed. Be
         aware that this can lead to very long simplification times.
 
+    dontcare : Boolean expression
+        Optimize expression under the assumption that this expression is false.
+        This is useful in e.g. Piecewise conditions, where later confitions
+        are subject to the previous conditions being false. Hence, if a previous
+        condition is And(A, B), the simplification of expr can be made with
+        don't cares for And(A, B).
```

Comment:
typo -> confitions.

Would it make more sense to negate this and call the argument something like `assumptions`? I think maybe it's easier to understand assuming something to be True rather than assuming something to be False.

Reply:
Or perhaps a better name for the argument would be `assume_false`.

Reply:
I changed the doc-string to better explain what is actually going on.

Reply:
Okay, that makes more sense.